docs/source/UserGuide/Tutorials/JobManagementAdvanced/index.rst

```diff
@@ -439,15 +439,19 @@ It can now be used together with the **dirac-wms-job-submit** command line tool.
 7.3.7 Submitting MultiProcessor (MP) jobs
 @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
 
-Jobs that can (or should) run using more than 1 processor should be described as such, using the "Tag" mechanism::
+Jobs that can (or should) run using more than 1 processor should be described as such,
+using the "setNumberOfProcessors" method of the API::
 
       j = Job()
       j.setCPUTime(500)
       j.setExecutable('echo',arguments='hello')
       j.setExecutable('ls',arguments='-l')
       j.setExecutable('echo', arguments='hello again')
       j.setName('MP test')
-      j.setTag('MultiProcessor')
+      j.setNumberOfProcessors(16)
+
+By calling Job().setNumberOfProcessors(), with a value bigger than 1,
```

Comment:
```suggestion
Calling ``Job().setNumberOfProcessors()``, with a value bigger than 1,
```

Reply:
I think this got lost again. The "By" also causes a grammar error with the "will translate" in the next line